src/dashboard/Push/PushIndex.react.js

```diff
@@ -145,7 +145,7 @@ let getPushName = (pushData) => {
       payload = JSON.parse(payload);
     } catch(e) { }
     if(payload){
-      return payload.alert ? payload.alert : payload;
+      return payload.alert ? payload.alert.toString() : payload;
```

Comment:
@drew-gross If `payload.alert` is an object then the push name will be  "[object Object]". Can we do the following instead to handle objects and strings:

```
return payload.alert ? JSON.stringify(payload.alert) : payload;
```